client/me/purchases/remove-purchase/index.jsx

```diff
@@ -163,14 +160,12 @@ const RemovePurchase = React.createClass( {
 			<div>
 				<p>
 					{
-						this.translate( 'Are you sure you want to remove %(productName)s from {{em}}%(siteSlug)s{{/em}}?', {
-							args: { productName, siteSlug: this.props.selectedSite.slug },
-							components: { em: <em /> }
+						this.translate( 'Are you sure you want to remove %(productName)s from {{siteName/}}?', {
+							components: { siteName: <em>{ this.props.selectedSite.slug }</em> }
 						} )
 					}
 					{ ' ' }
 					{ this.translate( 'You will not be able to reuse it again without purchasing a new subscription.', {
-						context: "Removal confirmation on Manage Purchase page",
```

Comment:
Yes, it's not very useful information for a translator. A context is meant to create the ability to translate two strings differently depending on the location of the original text. This text will not appear anywhere else, so any additional information to a translator is suited well in a comment (which already exists).